aicsimageio/readers/tiff_reader.py

```diff
@@ -194,7 +198,14 @@ def dims(self) -> str:
         if self._dims is None:
             # Get a single scenes dimensions in order
             with TiffFile(self._file) as tiff:
-                single_scene_dims = tiff.series[0].pages.axes
+                scenes = tiff.series
+                if not self._scene_shape_is_consistent(tiff, S=self.specific_s_index):
+                    scenes = [scenes[self.specific_s_index]]
+
+                single_scene_dims = scenes[0].pages.axes
+                if scenes[0].keyframe.samplesperpixel != 1:
+                    # if it's an RGB pixeltype then map Samples (S) to Channels (C)
+                    single_scene_dims = single_scene_dims.replace("S", "C")
```

Comment:
what if there is a multichannel image in which each channel contains RGB pixels?

Reply:
So I can't find an example file to test against. There would also be a mismatch between metadata and the data shape. I generally think acquiring a multichannel RGB file would be a bad idea. I think the current implementation will work but the metadata dimensions will need to be thought of in the light that it refers to RGB data.

Reply:
E.g. `tifffile.imwrite('two-channel-rgb.ome.tif', shape=(2, 32, 32, 3), dtype='uint8', photometric='rgb')`.
This will have an OME `DimensionOrder="XYCZT"` and `SizeC="6"`. Tifffile will return the data in dimension order `CYXS` because the stored data can be mapped directly to a contiguous numpy or zarr array without "swizzling" around dimensions. Bio-Formats "swizzles", tifffile doesn't.

Reply:
I think the way this library is constructed, on the reading side, we may have to either pack the RGB into a single intensity value or make them into channels (which seems incredibly messy to me because it would totally destroy the ability to report the correct number of channels from metadata).  The upper levels of the public api only expect to deal with single intensity per channel.   @heeler How does the non-dask code path return the data?